pkg/kp/config.go

```diff
@@ -41,6 +43,6 @@ func NewConsulClient(opts Options) *api.Client {
 	}
 
 	// error is always nil
-	ret, _ := api.NewClient(conf)
-	return ret
+	client, _ := api.NewClient(conf)
```

Comment:
this code is pulling a fast one on errcheck. Either that, or errcheck is able to read documentation ;)